python/lsst/sconsUtils/eupsForScons.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 try:
-    from eups import *
+    from eups import *  # noqa F403 F401 Importing everything into this namespace
```

Comment:
yes, it's so that all the normal eups routines can be available instead of the fallbacks. Since we are explicitly specifying fallbacks then it doesn't seem entirely crazy to import an explicit list from eups but is it worth it?